libgit2-sys/lib.rs

```diff
@@ -2246,7 +2246,14 @@ extern "C" {
         remote: *const git_remote,
     ) -> c_int;
     pub fn git_remote_get_refspec(remote: *const git_remote, n: size_t) -> *const git_refspec;
+
+    // This function is no longer used by kept here for the sake of completeness and to ensure
```

Comment:
Not sure about the wording here?

```suggestion
    // This function is no longer used but is kept here for the sake of completeness and to ensure
```

Also, I'm not really sure this comment is necessary. There are quite a few other functions that I believe are no longer used. We don't do any sort of auditing of what is missing or not.

Reply:
@ehuss I have removed the comments and actioned the changes requested below. Please take a look.

I still think there is scope for improvement of the API by propagating errors but I'm assuming that can only be done with a major release. LMK if you want me to create a ticket for this?